plugin/src/main/groovy/org/openbakery/appcenter/AppCenterPluginExtension.groovy

```diff
@@ -13,35 +13,27 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.openbakery.hockeyapp
+package org.openbakery.appcenter
 
 import org.gradle.api.Project
 
 
-class HockeyAppPluginExtension {
-	def Object outputDirectory
-	def String apiToken = null
-	def String appID = null
-	def String notes = "This build was uploaded using the gradle xcodeplugin"
-	def String status = 2
-	def String notify = 1
-	def String notesType = 0
-	def String[] tags = null
-	def String[] teams = null
-	def String[] users = null
-	def String releaseType = null
-	def String mandatory = 0
-	def String privatePage = false
-	def String commitSha = null
-	def String buildServerUrl = null
-	def String repositoryUrl = null
+class AppCenterPluginExtension {
+	Object outputDirectory
+	String appOwner = null
+	String appName = null
+	String apiToken = null
+	String destination = null
```

Comment:
Collaborators is the standard usergroup, have you considered to make this as default. Right now there is no error message and it does not fail, only a log `App Center: IPA upload distributed to: NULL`

Reply:
sorry I haven't pulled and used the commit from Sept 13th

Reply:
Great idea and thank you!  I just set the default group to "Collaborators" in the latest commit.